Source.lua

```diff
@@ -531,7 +529,7 @@ local function luau_load(module, env, luau_settings)
 		luau_validatesettings(luau_settings)
 	end
 
-	if type(module) == "string" or type(module) == "buffer" then
+	if not (type(module) == "table") then
```

Comment:
use `if type(module) ~= "table" then`
Having a `not` is one extra unnecessary operation.

Reply:
not me forgetting that != not existing doesn't mean ~= doesn't exist